queries/tsx/textobjects.scm

```diff
@@ -1 +1 @@
-; inherits: typescript,jsx
```

Comment:
But typescript should include JS. Can you explain what is not working with the current tsx queries?

Reply:
@theHamsta Absolutely nothing is working with the current `tsx` queries, I don't understand at all why.

I tried ` ; inherits (typescript)` also and that didn't work.

However `; inherits (javascript)` works great.

When you look at `typescript/textobjects.scm` it's doing `; inherits (javascript)` also, and in turn, `javascript` is already inheriting `jsx`.

I thought maybe `; inherits typescript,jsx` might be breaking because `typescript` already imports `javascript` which imports `jsx` leading to a double import of `jsx`, but then why does `; inherits (typescript)` also not work.

I'm thinking, maybe something is fundamentally broken with the inheritance system? As if transitive inheritance isn't working?

Reply:
@theHamsta I guess it's also important to note that `tsx` is the *only* `textobjects.scm` that uses transitive inheritance so it might well make sense that transitive inheritance doesn't work.

In that case maybe this is a pragmatic change to make until transitive inheritance can be fixed (maybe you don't even need to support transitive inheritance?, in this case it's pretty much redundant given `; inherits: (javascript)` is already completely equivalent)

Reply:
can you remove the parenthesis in `typescript/textobjects.scm` it's working fine for me. There have been some problems with JS/TS having different parameter nodes but this problems does not seem to be triggered with our current queries. The parenthesis mean that this language is not automatically transitioned to indirect dependencies.

Reply:
Cool I'll try that.

Reply:
@theHamsta Great, thanks for the advice, that works! Updated this PR and everything is looking good.